niet/__init__.py

```diff
@@ -127,6 +130,44 @@ def get(data, keywords, silent=False):
         sys.exit(1)
 
 
+def delete_object(indict, path):
```

Comment:
Can you create a new module file named `actions` where you will move this one and rename it `delete`, to avoid to store everything in this module and keep it clean.

So in a second time you will import your new created module `import niet.actions` and call your function like this:
```python
niet.actions.delete(result, to_delete)
```